test/support/coronavirus_landing_page_steps.rb

```diff
@@ -166,16 +166,16 @@ def then_i_can_see_the_nhs_banner
   def then_i_can_see_the_timeline
     assert page.has_selector?("h2", text: "Recent and upcoming changes")
 
-    assert page.has_selector?(".covid-timeline__item-heading", text: "18 September")
+    assert page.has_selector?("#covid-timeline h3", text: "18 September")
```

Comment:
Thinking about the other conversation about testing and IDs, should this be swapped to class in this instance?